packages/internals/utils/src/utils.js

```diff
@@ -23,9 +23,6 @@ const utils = module.exports = {
         if (!_.isPlainObject(sourceValue) || !_.isPlainObject(objectValue)) {
           return sourceValue;
         }
-        if (_.isUndefined(sourceValue)) {
-          return objectValue;
-        }
```

Comment:
What's the reason behind removing this? As far as I remember it was needed to ensure that an undefined value in the source object has it's value correctly filled by the equivalent property value in the defaults object. Or am I missing something?